pkg/operator/sync_controllermanager_v311_00.go

```diff
@@ -271,7 +271,9 @@ func manageServiceCatalogControllerManagerDeployment_v311_00_to_latest(
 	}
 
 	// if trustedCAModified we should add a mount point to the daemonset
-	if trustedCAModified {
+	// also if there is a daemonset with a trusted mountpoint, we need to ensure
+	// we keep it.
+	if trustedCAModified || daemonsetHasTrustedCAVolume(client, required.Namespace, required.Name) {
```

Comment:
What happens if someone comes along and deletes the trusted CA volume mount and/or volume from the `DaemonSet` without any modifications happening on the trusted CA config map? Will those get put back automatically?

Seems like the check should really just be:
```go
if trustedCAConfigMapExists(...) {
    addTrustedCAVolumeToDaemonSet(required)
}
```

Reply:
@joelanford @shawn-hurley Okay, I made the change to look if the configmap exists. I then deployed it, deleted the volume from the daemonset, and saw the operator put it back. And confirmed that the configmap existed.